package.json

```diff
@@ -28,27 +28,36 @@
   "devDependencies": {
     "@commitlint/types": "11.0.0",
     "@types/jest": "26.0.15",
-    "@types/node": "12.12.37",
-    "@typescript-eslint/parser": "4.6.0",
-    "@vercel/ncc": "0.24.1",
-    "eslint": "7.12.1",
-    "eslint-plugin-github": "3.4.1",
-    "eslint-plugin-jest": "23.20.0",
+    "@types/node": "14.14.7",
+    "@typescript-eslint/parser": "4.8.1",
+    "@vercel/ncc": "0.25.1",
+    "eslint": "7.13.0",
+    "eslint-plugin-github": "4.1.1",
+    "eslint-plugin-jest": "24.1.3",
     "eslint-plugin-prettier": "3.1.4",
-    "jest": "25.5.3",
-    "jest-circus": "25.5.3",
+    "jest": "26.6.3",
+    "jest-circus": "26.6.3",
     "js-yaml": "3.14.0",
     "prettier": "2.1.2",
-    "ts-jest": "25.4.0",
-    "typescript": "3.9.7"
+    "ts-jest": "26.4.4",
+    "typescript": "4.0.5"
   },
   "dependencies": {
-    "@actions/core": "1.2.6",
-    "@actions/github": "4.0.0",
-    "@commitlint/lint": "11.0.0",
-    "@commitlint/load": "11.0.0"
-  },
-  "resolutions": {
-    "import-fresh": "JulienKode/import-fresh#fix/issue-when-there-no-parent-module"
+    "@actions/core": "^1.2.6",
+    "@actions/github": "^4.0.0",
+    "@commitlint/config-angular": "^11.0.0",
+    "@commitlint/config-conventional": "^11.0.0",
+    "@commitlint/config-lerna-scopes": "^11.0.0",
+    "@commitlint/config-patternplate": "^11.0.0",
+    "@commitlint/format": "^11.0.0",
+    "@commitlint/lint": "^11.0.0",
+    "@commitlint/load": "^11.0.0",
+    "commitlint-config-jira": "^1.4.1",
+    "commitlint-plugin-jira-rules": "^1.4.0",
+    "conventional-changelog-lint-config-canonical": "^1.0.0",
```

Comment:
Currently we have limited number of commit lint module because:
- we load the config
- we run the lint 

Then has we are just doing that, every people can use this library as they want. If they need another commit lint module they can just install the dependencies that they want at the version that they want